rawtransactions.go

```diff
@@ -552,7 +552,9 @@ func (r FutureSearchRawTransactionsResult) Receive() ([]*wire.MsgTx, error) {
 func (c *Client) SearchRawTransactionsAsync(address btcutil.Address, skip, count int) FutureSearchRawTransactionsResult {
 	addr := address.EncodeAddress()
 	verbose := btcjson.Int(0)
-	cmd := btcjson.NewSearchRawTransactionsCmd(addr, verbose, &skip, &count)
+	prevOut := btcjson.Int(0)
```

Comment:
Hmmm.  I typically like to be explicit in case the default changes at some point, but I can change it.